### DISCUSSION TITLE
Delete Observation

### DISCUSSION THREAD

**Discussion by slussinator** [olivierfriard/BORIS#460] [Q&A]
I have uploaded several observations outside of a project. Now I can't seem to delete them. Also when starting a new projects I am unable to add these observations to the project. 
When I select observations -> edit observation -> and then try to remove the observation file I receive the message  "A media file must be loaded in #1". I can't figure out how to delete the observation alltogether.

Thank you in advance!

**Comment by olivierfriard** [comment]
Hi,

RTMF...
https://boris.readthedocs.io/en/v7/#delete-observations

Cheers

Olivier
